src/server/console/logind_console_services.cpp

```diff
@@ -119,9 +122,9 @@ std::unique_ptr<LogindSession, decltype(&g_object_unref)> simple_proxy_on_system
 }
 
 std::unique_ptr<LogindSeat, decltype(&g_object_unref)> simple_seat_proxy_on_system_bus(
-    mir::GLibMainLoop& ml,
-    GDBusConnection* connection,
-    char const* object_path)
+    mir::GLibMainLoop &ml,
+    GDBusConnection *connection,
+    char const *object_path)
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style